client/stylesheets/slick.css

```diff
@@ -1,4 +1,14 @@
 /* Slider */
+
+img {
+  max-width: 253px !important;;
+  max-height: 176px !important;
+}
+
+.minHei {
+  min-height: 175px !important;
+}
```

Comment:
It seems you're using these styles in news. We have a separate file for these styles.